### DISCUSSION TITLE
Proposal: Giscourse -- a forum system using Giscus as backend

### DISCUSSION THREAD

**Discussion by expikr** [giscus/giscus#1592] [Ideas]
GitHub discussions basically provides a complete forum system, however its UI sucks whereas Giscus is amazing.

It would be cool to expand Giscus or create a standalone product to create a dynamic forum on on GitHub Pages with your own html frontend acting as a thin fetcher to the GitHub Discussions API.

**Comment by laymonage** [comment]
Cool idea, but I don't think it's in giscus' scope. If the code is useful, feel free to fork the repo to build the feature. (Make sure to host it separately instead of using the giscus.app API though 🙂)
